### PR TITLE
Changed comments /**/ to //, removed deprecated @silent

### PR DESCRIPTION
**Pull request by Anahkiasen**
Just a little pull request on the issue I opened yesterday. I changed the comments with a regex so it may not be perfect, but it should help you guys a little already.

I also removed an @silent in icon-font, it's a deprecated directive that was replaced by %, so I replaced it.

### CODE REVIEW

**Comment by MoOx** [comment]
First of all, thanks a lot for you contribution !
The only problem I see with your changes, is that now we cannot determine what comment blocks are usable for an auto doc 
The thing with /*\* */ is it's easy "found-able". But now it's harder.
I thing we can remplace like this

```
/** => //**
 *  => //
 */ => //*
```

Maybe it's risky to use //*, so we can use

```
/** => ////
 *  => //
 */ => ///
```

What do you think of that kind of update ? /cc @canarymason

Btw, can you please write down the regex you've used ?

Again, thanks !

File: stylesheets/recipes/_color.scss [hunk 1]
```diff
@@ -1,21 +1,10 @@
-/**
- *
- * @author David Kaneda - http://www.davidkaneda.com
- *
- */
+//  @author David Kaneda - http://www.davidkaneda.com
```
Comment (MoOx) [inline stylesheets/recipes/_color.scss:6]: You see, here you now just have a simple line, instead of a usable block for doc.

**Comment by Anahkiasen** [comment]
If you're creating the doc-generation code yourself and can use any syntax for it, I can redo another regex to fit your needs.
I'll redo a pull request.

**Comment by Anahkiasen** [comment]
I actually ended up doing something even simpler.

```
// /**
//  * Blueprint grid background pattern
//  *
//  * @link http://lea.verou.me/css3patterns/#blueprint-grid
//  *
//  * @author Lea Verou http://lea.verou.me/ for the original pattern
//  * @author Maxime Thirouin maxime.thirouin@gmail.com @MoOx for the sass mixin
//  */
```

Simple as that.
`(\n|^)(/\**| ?\*| \*/)( )?` => `$1// $2$3`
I also added a missing @import in effect.scss.

**Comment by MoOx** [comment]
Hum, maybe it's a bit weird now.
Can we just use something like this instead ? More simpler to read

```
/** => ////
 *  => //  
 */ => ///
```

I think this need a little reflection.

/ Ping @davidkaneda (which is working on a doc generator for sass mixins)

**Comment by codingdesigner** [comment]
I'm glad to see more movement on this. I didn't get a chance to do much with this yesterday or today, so I'll close my branch for fixing comments. 

FWIW, I favor a pattern that just uses different numbers of /, like MoOx's last comment.

**Pull request closed without merging**